crates/turborepo-ui/src/tui/app.rs

```diff
@@ -137,8 +137,10 @@ impl<W> App<W> {
     pub fn start_task(&mut self, task: &str) -> Result<(), Error> {
         // Name of currently highlighted task.
         // We will use this after the order switches.
-        let highlighted_task =
-            &self.tasks_by_status.task_names_in_displayed_order()[self.selected_task_index];
+        let highlighted_task = self
+            .tasks_by_status
+            .task_name(self.selected_task_index)
+            .to_string();
```

Comment:
We need to clone this as otherwise we end up holding an immutable reference to `App` which would prevent us from mutating `App` to move the task from planned into starting.